designsafe/static/scripts/projects/components/manage-simulations/manage-simulations.template.html

```diff
@@ -7,6 +7,14 @@ <h4 class="modal-title" style="border:none;">
     </h4>
 </div>
 <div class="modal-body">
+    <div style="margin-bottom: 20px;">
+        <span style="font-weight: bold;">
+            Simulation: </span>
+        <span>
+            A stand-alone dataset published under a unique citation and DOI. 
+            One or more can be published under a project and each can be published progressively and versioned.
```

Comment:
Versioning an individual simulation isn't supported, so we should remove this language.